README.md

```diff
@@ -82,6 +83,17 @@ docker run -p 8000:8000 wagtailcms
 
 ```
 
+## Environment variables
+
+| Variable        | Default       | Description                       |
+| --------------- | :-----------: | --------------------------        |    
+| DBHOST         | unistatsdb.postgres.database.azure.com            | DB host url/string                |
```

Comment:
Default values should be those used to run the CMS locally not in the development environment, please change this default values to something like the following:

```
DBNAME - <database name>
DBHOST - 127.0.0.1
DBPORT - 5432
DBUSER - <username>
DBPASSWORD - <password>
```